cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -296,5 +289,23 @@ func (command *Evaluate) Execute(args []string) (err error) {
 		return nil
 	})
 
+	if err := writeCSVs(command.ResultPath, assessments); err != nil {
+		log.Panicf("ERROR: %s", err)
+	}
+
+	return nil
+}
+
+// WriteCSVs writes the various CSV reports to disk.
+func writeCSVs(resultPath string, assessments report.AssessmentPerModelPerLanguagePerRepository) (err error) {
```

Comment:
nitpick

this change does not really have anything to do with the newly introduced interface of this commit

Reply:
so should maybe go into the next one?

Reply:
Well due to the change to an Interface I needed to revise how the `evaluation.csv` is written that is why it is in there.